backend/dal/Services/Concrete/PropertyService.cs

```diff
@@ -141,7 +142,20 @@ public IEnumerable<PropertyModel> Search(AllPropertyFilter filter)
             }
 
             var query = this.Context.GenerateAllPropertyQuery(this.User, filter);
-            var properties = query.Select(p => p.PropertyTypeId == Entities.PropertyTypes.Land ? new ParcelModel(p, this.User) as PropertyModel : new BuildingModel(p, this.User)).ToArray();
+            var properties = query.Select(p => new[] { Entities.PropertyTypes.Land, Entities.PropertyTypes.Subdivision }.Contains(p.PropertyTypeId) ? new ParcelModel(p, this.User) as PropertyModel : new BuildingModel(p, this.User)).ToArray();
```

Comment:
This code is reused from another area of the application to get the workflow code. Note that this code doesn't address the case where there are multiple projects. We need some logic that can determine the most advanced project workflow and return that to the frontend.

Reply:
@Fosol maybe not out of scope. with this logic, if the property is in an SPP project and a draft project the project workflow will be set to the workflowCode of DRAFT and then won't display properly on the map (depending on the order of the projects in the DB).